app/scripts/lib/constants.js

```diff
@@ -41,6 +41,8 @@ define(function (require, exports, module) {
 
     SYNC_SERVICE: 'sync',
 
+    SYNC11_MIGRATION: 'sync11',
+    AMO_MIGRATION: 'amo',
```

Comment:
@vladikoff - yeah, let's keep it specific for now. We can expand in the future if needed.